### PR TITLE
Sitemap overflow fix

### PR DESCRIPTION
**Pull request by DarkSilence**
That's my implementation for a fix for issues #82 and #81 
I'm not quite sure about my approach on line 202. Could you check it please.

### CODE REVIEW

File: Sitemap.php [hunk 1]
```diff
@@ -166,7 +171,7 @@ private function createNewFile()
         // Use XML stylesheet, if available
         if (isset($this->stylesheet)) {
             $this->writer->writePi('xml-stylesheet', "type=\"text/xsl\" href=\"" . $this->stylesheet . "\"");
-            $this->writer->writeRaw("\n");            
+            $this->writer->writeRaw(PHP_EOL);
```
Comment (samdark) [inline Sitemap.php:174]: That would result in CRLF in generated sitemaps under Windows. That was originally not not desirable. Is there a reason for the change?

Comment (samdark) [inline Sitemap.php:174]:
```suggestion
            $this->writer->writeRaw("\n");
```

File: Sitemap.php [hunk 2]
```diff
@@ -180,21 +185,22 @@ private function createNewFile()
          * the header was written correctly and we can simply reuse any <url>
          * elements that did not fit into the previous file. (See self::flush)
          */
-        $this->writer->text("\n");
-        $this->flush(true);
+        $this->writer->text(PHP_EOL);
```
Comment (samdark) [inline Sitemap.php:188]:
```suggestion
        $this->writer->text("\n");
```

**Comment by mougrim** [comment]
@samdark @DarkSilence, Hi!
What is needed for merge fix? Do you need some help here?
I have the same issue:
```
PHP Fatal error:  Uncaught OverflowException: The buffer size is too big for the defined file size limit in vendor/samdark/sitemap/Sitemap.php:234
Stack trace:
#0 vendor/samdark/sitemap/Sitemap.php(199): samdark\sitemap\Sitemap->flush()
#1 vendor/samdark/sitemap/Sitemap.php(212): samdark\sitemap\Sitemap->finishFile()
```

**Comment by samdark** [comment]
Yes.

Need to verify it works and works correctly.

**Comment by mougrim** [comment]
Hello!
Looks like it doesn't fix OverflowException, I'm preparing separated MR.

**Pull request closed without merging**